packages/eslint-plugin-nx/src/utils/runtime-lint-utils.ts

```diff
@@ -419,16 +419,22 @@ export function isAngularSecondaryEntrypoint(
   importExpr: string
 ): boolean {
   const targetFiles = targetProjectLocator.findPaths(importExpr);
```

Comment:
The current situation is:
- Linter already knows the target project when it calls the `findPaths` so steps 1-2 are already covered earlier
- Based on import we need to find the file to which the import maps so that we can traverse back to see if it belong to secondary entry point

Proposal 1:
- There might be several `ng-package.json`. Unless we know where exactly import points to in the file tree we cannot find out whether it belongs to secondary entry point (and to which one)

Proposal 2:
- This might work using `typescript.resolveModuleName` but it would be a somewhat expensive operation.

Reply:
The proposal 2 works fine